src/point_one/fusion_engine/messages/defs.h

```diff
@@ -414,6 +414,11 @@ enum class Response : uint8_t {
    * The device is in a state where it can't process the command.
    */
   UNAVAILABLE = 9,
+  /**
+   * A TransportType specified in the command is invalid, or unsupported on the
+   * target device.
+   */
+  TRANSPORT_TYPE_ERROR = 10,
```

Comment:
I'd suggest we call this `UNSUPPORTED_INTERFACE` instead. I think that's a little more clear since this error code should apply for both an unsupported transport (you asked for TCP on a serial-only device, etc.) and also a bad index (you asked for serial port 2000 and there are only 2). Essentially, this applies if any of the fields in the `InterfaceID` structure are wrong, not just the transport type.

Unless you want to have 2 new, separate error codes for invalid index and transport type, where transport type would just be that you gave a valid enum, but one that isn't supported on this type of device? I think invalid enum would just fall under `VALUE_ERROR`.